src/Mappers/PluginMapper.php

```diff
@@ -51,7 +51,7 @@ private function getPluginDefinition(string $handle, array $pluginInfo): array
         return [
           'isEnabled' => $pluginInfo['isEnabled'],
           'isInstalled' => $pluginInfo['isInstalled'],
-          'settings' => $settings ? $settings->attributes : [],
+          'settings' => $settings->attributes ?? [],
```

Comment:
Won't this break if no plugin can be found with `Craft::$app->plugins->getPlugin($handle)` ?